docs/topics/data-analysis/data-science-libraries.md

```diff
@@ -1,88 +1,43 @@
+| [Tablesaw](https://github.com/jtablesaw/tablesaw)                      | - Data collection <br/> - Data cleaning and processing <br/> - Data exploration and visualization | - Tools for loading, cleaning, transforming, filtering, and summarizing data <br/> -  Inspired by [Plot.ly](https://plotly.com/)                                                                                                                                                                                                                                                                            |
+| [Dex](https://github.com/PatMartin/Dex)                                | - Data collection <br/> - Data cleaning and processing <br/> - Data exploration and visualization | - Tools for reading, transforming, and visualizing data <br/> - Written in Groovy and Java on top of [JavaFX](https://openjfx.io/)                                                                                                                                                                                                                                                                          |
+| [CoreNLP](https://stanfordnlp.github.io/CoreNLP/)                      | Data cleaning and processing                                                                      | - Natural language processing toolkit <br/> - Linguistic annotations for text, such as sentiment and quote attributions <br/> - Support for eight languages                                                                                                                                                                                                                                                 |
+| [Smile](https://github.com/haifengl/smile)                             | - Data cleaning and processing <br/> - Data exploration and visualization <br/> - Model building  | - Ready-made algorithms for machine learning and natural language processing <br/> - Linear algebra, graph, interpolation, and visualization tools <br/> - Smile provides a functional [Kotlin API](https://github.com/haifengl/smile/tree/master/kotlin), [Scala API](https://github.com/haifengl/smile/tree/master/scala), [Clojure API](https://github.com/haifengl/smile/tree/master/clojure), and more |
+| [Smile-NLP-kt](https://github.com/londogard/smile-nlp-kt)              | Data cleaning and processing                                                                      | - Kotlin rewrite of the [Scala](https://www.scala-lang.org/api/current/) implicits for the natural language processing part of Smile <br/> - Operations in the format of Kotlin extension functions and interfaces <br/> - Sentence breaking, stemming, bag of words, and other tasks                                                                                                                       |
```

Comment:
Note: The last code update for this library was 4 years ago

Reply:
Let's keep